combat-trainer.lic

```diff
@@ -1816,6 +1819,12 @@ class TrainerProcess
       pickbox(game_state)
     when 'Hunt'
       bput('hunt', 'You take note of ', 'You find yourself unable to hunt in this area') unless game_state.retreating?
+    when 'Teach'
+      UserVars.friends.each do |character|
```

Comment:
💡 Should this be a setting that gets merged with a UserVar in dependency.lic? Or are you confident people would only want to use a UserVar here?

Reply:
that uservar is populated by hunting_buddies: in your yaml, I don't really see value in making it separately configurable outside of that since you want to be hunting with them for it to work

Reply:
:+1: I missed that, thanks